v2/discoveryv1/discovery_v1_suite_test.go

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2018, 2021.
+ * (C) Copyright IBM Corp. 2021.
```

Comment:
oh I wonder if you need to run lint on the copyright again?

Reply:
@nan2iz yep good catch. I just updated them.